x-pack/plugins/infra/public/pages/metrics/layouts/host.ts

```diff
@@ -128,32 +224,65 @@ export const hostLayoutCreator: InfraMetricLayoutCreator = theme => [
       {
```

Comment:
Shouldn't the "Kubernetes" label above be translated under the `xpack.infra.metricDetailPage.kubernetesMetricsLayout.layoutLabel` id?

Reply:
According to [Glossary](https://docs.google.com/document/d/1e-lHAPAtxodP8U4gbwA3lw8FyhhW7M303--77nMpe8w/edit) we shouldn't translate this label.

Reply:
:+1: didn't know that - sorry